### DISCUSSION TITLE
Single Click in FileSystem

### DISCUSSION THREAD

**Discussion by Chevifier** [godotengine/godot-proposals#7910] [Editor]
Add an option to single click resources and scene in the filesystem to open them.

![Screenshot from 2023-09-28 22-38-33](https://github.com/godotengine/godot-proposals/assets/9784593/92070564-0a52-469b-ba84-eb143932d57a)

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/1699.
